### ISSUE TITLE
Handle basic natural language queries

### ISSUE THREAD

**Issue by DonaldKellett** [codewars/discord-bot#23]
As suggested [on Discord](https://discord.com/channels/846624424199061524/852248931970973716/932984987886694461), what if our bot had a command that could process basic natural language queries and maybe respond to them by quoting from the appropriate [Codewars docs](https://docs.codewars.com), and / or attempt to translate them into other standard commands? E.g. assuming my server nick matches my Codewars username, I could ask:

> ?ask How many 1 kyus do I need to complete to reach 2 dan?
> 
> 30

Basically something similar to [this assignment in an AI course](https://cs50.harvard.edu/ai/2020/projects/6/questions/), which doesn't seem _too_ sophisticated, though perhaps a more complex approach is required for greater accuracy. Of course, there's also the question of whether such things are suitable in Node.js / TypeScript compared to Python.

What do you think?
